### DISCUSSION TITLE
Civil tools

### DISCUSSION THREAD

**Discussion by Paolo-lab** [iTwin/itwinjs-core#7466] [Q&A]
Hello everyone, good morning, I wanted to understand if there is the possibility of implementing in our civil tools viewer. I attach an image taken from the original viewer.
<img width="346" alt="CivilTools" src="https://github.com/user-attachments/assets/ac8a566c-75ba-4209-bb1c-4812ce3c021e">

I did not find any material or examples, but I read the thread #3864 in which the possibility of a future development was aired

Thanks Paul

**Comment by markschlosseratbentley** [comment]
@aruniverse may have an idea.
